File: go/orm/internal/conformance/amendfixture/V0002.go
package amendfixture

import (
	"github.com/kosmassp/SimpleOrm/go/orm"
	amendwidget "github.com/kosmassp/SimpleOrm/go/orm/internal/conformance/amendfixture/Table/AmendWidget"
)

type V0002 struct{}

func (V0002) Compose(version *orm.VersionBuilder) {
	version.Apply(amendwidget.V0002_AddNote{})
}

File: php/tests/Conformance/AmendFixture/V0002.php
<?php

declare(strict_types=1);

namespace SimpleOrm\Tests\Conformance\AmendFixture;

use SimpleOrm\Migrations\MigrationVersion;
use SimpleOrm\Migrations\VersionBuilder;
use SimpleOrm\Tests\Conformance\AmendFixture\Table\AmendWidget\V0002_AddNote;

final class V0002 extends MigrationVersion
{
    public function compose(VersionBuilder $version): void
    {
        $version->apply(V0002_AddNote::class);
    }
}
